### PR TITLE
add capacities to override some method of the plugin

### PR DESCRIPTION
**Pull request by TisserantG**
…de in some case

### CODE REVIEW

**Comment by Aschen** [comment]
Can you add a more explicit title for the PR please?

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "kuzzle-device-manager",
-  "version": "0.4.14",
+  "version": "0.4.16",
```
Comment (Aschen) [inline package.json:3]: This should be done automatically by the release tool

**Review by Aschen — CHANGES_REQUESTED** [review]

**Comment by TisserantG** [comment]
> Can you add a more explicit title and proper label for the PR please?
Done

File: package.json (hunk 1 above)
Comment (TisserantG) [reply package.json]: done

File: package.json [hunk 2]
```diff
@@ -25,29 +25,30 @@
   "license": "Apache-2.0",
   "dependencies": {
     "csvtojson": "~2.0.10",
+    "kuzzle": "^2.18.0",
```
Comment (Aschen) [inline package.json:28]: kuzzle should be left as peerDependency only
```suggestion
```

**Review by Aschen — APPROVED** [review]

**Pull request merged**